tools/percolator/rollout_nested_collection.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="rollout_nested_collection" name="Rollout nested list" version="3.1">
+    <description>into a single long list</description>
```

Comment:
Nah, think that was about the nested-list creator in this repo. But I didn't think to look further when I suddenly figured I also needed this yesterday evening. Flatten seems to do exactly what I need so I will remove my tool!